lib/src/html_stylist.dart

```diff
@@ -59,7 +59,8 @@ class HTML {
   static TextSpan toTextSpan(BuildContext context, String htmlContent,
       {Function(dynamic)? linksCallback,
       Map<String, TextStyle>? overrideStyle,
-      TextStyle? defaultTextStyle}) {
+      TextStyle? defaultTextStyle,
+      bool unescapeContent = true}) {
```

Comment:
Since we're changing it - how about adding a trailing comma (same for the other method)? 😊

Reply:
Done!